src/index.js

```diff
@@ -0,0 +1,187 @@
+state = {
+    temp: 50,
+    city: "Atlanta",
+    latitude: 0,
+    longitude: 0
+};
+
+PATH = "http://127.0.0.1:5000/weather"
+const findWeather = () => {
+    
+    axios.get(PATH,
+        {
+            params: {
+                "lat": state.latitude,
+                "lon": state.longitude
+            }
+        })
+        .then((response) => {
+            let tempK = response.data.main.temp
+            state.temp = Math.round(((tempK-273.15)*1.8) + 32)
+            console.log(response.data.main.temp)
+            const tempNumContainer = document.querySelector("#currentTemp")
+            tempNumContainer.textContent = `${state.temp}°`;
+            colorChanging();
+            changeGround();
+        })
+        .catch((error) => {
+            console.log('error in findWeather for', state.city);
+            throw error;
+        })
+    }
+
+WeatherPATH = "http://127.0.0.1:5000/location"
+const findLatitudeAndLongitude = () => {
+    
+    axios.get(WeatherPATH,
+        {
+            params: {
+                "q": state.city
+            },
+        })
+        .then((response) => {
+            state.latitude = response.data[0].lat;
+            state.longitude = response.data[0].lon;
+            console.log('success', state.city, state.latitude, state.longitude);
+            findWeather();
+            colorChanging();   
+            changeGround();
+         })
+        .catch((error) => {
+            console.log('error in findLatitudeAndLongitude for', state.city);
+            throw error;
+        })
+    }
+
+
+const changeGround = function () {
+    if (state.temp <= 39) {
+        document.querySelector("#currentTemp")
+        const tempGroundContainer = document.querySelector("#groundChange")
+        tempGroundContainer.textContent = `❄️🌲❄️☃️❄️🏂❄️🌲❄️`
+    }
+
+    else if (state.temp >= 40 && state.temp <= 59) {
+        document.querySelector("#currentTemp")
+        const tempGroundContainer = document.querySelector("#groundChange")
+        tempGroundContainer.textContent = `🌬️ 🍂 🦃 🌾 🍂 🌰 🐿️ 🌾 🍂`
+    }
+
+    else if (state.temp >= 60 && state.temp <= 69){
```

Comment:
Great function naming, this looks really clean 💯 Your conditonal comprehension is very strong